### DISCUSSION TITLE
Serialization build

### DISCUSSION THREAD

**Discussion by PaulaJLR** [RosettaCommons/rosetta#90] [PyRosetta] [answered]
Dear PyRosetta developers,
I had previously installed PyRosetta as described [here](https://www.pyrosetta.org/downloads#h.iwt5ktel05jc), and it worked perfectly, however I am implementing multiprocessing in my script, so I am trying to install the serialization build. 
It seems to try to download the wheel from [this link](https://west.rosettacommons.org/pyrosetta/release/release/PyRosetta4.Release.python311.linux.cxx11thread.serialization.wheel/) but it does not exist, so I get:

```
Installing PyRosetta:
 os: linux
 type: Release
 Rosetta C++ extras: cxx11thread.serialization
 mirror: https://west.rosettacommons.org/pyrosetta/release/release
 extra packages: numpy

Could not retrive latest.html from 'https://west.rosettacommons.org/pyrosetta/release/release/PyRosetta4.Release.python311.linux.cxx11thread.serialization.wheel/' error-code: 404, aborting
``` 
Is there another way to install it, or could you please update the link?
Thank you so much,
Paula

**Comment by lyskov** [comment]
@PaulaJLR thank you for letting us know! Could you please tell me which Python version you are using for this? Thanks,

**Reply by PaulaJLR** [reply]
Thank you! I am using 3.11.5

**Reply by lyskov** [reply]
We have not build binaries for this combination of os/python/flags before so i added this to the queue. It should be ready within next day or two, - i will ping you when this is done.

**Reply by lyskov** [reply]
Ok, binaries is now ready, could you please give them a try? For next few days please use use `mirror=1` when calling the installer (binaries will get synced to a default server in a couple of days). Please let me know how it goes.
Thanks,

**Reply by PaulaJLR** [reply, marked as answer]
Thank you so much, installation works with
```bash
python -c 'import pyrosetta_installer; pyrosetta_installer.install_pyrosetta(serialization=True, skip_if_installed=False, mirror=1)'
```
and my script works as well.
I appreciate the prompt help!
Best regards,
Paula
